ocaml/xe-cli/newcli.ml

```diff
@@ -305,12 +305,14 @@ let exit_status = ref 1
 
 let with_open_tcp_ssl server f =
   let port = get_xapiport true in
+  let verify_cert = None in
+  (* CL *)
   debug "Connecting via stunnel to [%s] port [%d]\n%!" server port ;
   (* We don't bother closing fds since this requires our close_and_exec wrapper *)
   let open Safe_resources in
   Stunnel.with_connect ~use_fork_exec_helper:false
     ~write_to_log:(fun x -> debug "stunnel: %s\n%!" x)
-    ~extended_diagnosis:(!debug_file <> None) server port
+    ~verify_cert ~extended_diagnosis:(!debug_file <> None) server port
```

Comment:
I'd like to have a test in the Makefile / CI to track the number on usages where verify_cert is None. We really want to have at most a couple of this for the justified use-cases.

Can this be changed to `~verify_cert:None`? It would be greppable and easy to count the usages.

See https://github.com/xapi-project/xen-api/commit/b5f213f022f63d629c036411a51f68a25a7577b1#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52 for an example on how it could be integrated.

Reply:
I've made this more explicit

Reply:
Nice, thanks! could you add a step to test it to .github/workflows/main.yml?